### ISSUE TITLE
Plot over microbial samples: list samples in alphabetical order.

### ISSUE THREAD

**Issue by AnnaLeinfelt** [Clinical-Genomics/vogue#186]
**Who: Does your feature request involves multiple parties?**
A clear description of who is involved in this feature request. It can range from developers, assignees, benefitting parties,....

Can the microbial samples be in alphabetical order? 
 [In this plot](https://vogue.scilifelab.se/prepps/microbial/2020)

**Scope**
Add proper label based on the scope of this feature request, e.g. backend, frontend, bug fix, etc.

I think this is frontend!

**Please describe the problem that this feature request can solve.**
A clear and concise description of what the problem is.

Easier to get an overview of the samples if they are in alphabetical order. For instance, does all Klebsiella samples behave the same?
Background: Plot was requested by lab to be able to see if there were any trend between different species.

**Describe the solution you'd like**
A clear and concise description of what you want to happen or how to solve this problem.

To order the microbial samples in this plot in alphabetical order. 
 [Microbial samples plot](https://vogue.scilifelab.se/prepps/microbial/2020)

### Optional:
**Describe alternatives you've considered**

**Additional context**

**Comment by hassanfa** [comment]
@mayabrandi Could you have a look at this and see how much work is needed?

**Comment by mayabrandi** [comment]
Do you mean this plot @AnnaLeinfelt https://vogue.scilifelab.se/prepps/microbial/
Do you mean the organisms should be in alphabetic order?

**Comment by AnnaLeinfelt** [comment]
> Do you mean this plot @AnnaLeinfelt https://vogue.scilifelab.se/prepps/microbial/ Do you mean the organisms should be in alphabetic order?

Yes! Then it is easier to have an overview. (Adding a screenshot)
<img width="1180" alt="Screenshot 2021-11-29 at 10 41 32" src="https://user-images.githubusercontent.com/37171709/143844567-399e8a8c-699a-4797-a66c-082617b9f5a3.png">
